cmake/modules/DefineCMakeDefaults.cmake

```diff
@@ -23,12 +23,4 @@ set(GENERIC_LIB_VERSION "0.1.0")
 set(GENERIC_LIB_SOVERSION "0")
 
 # set -Werror
-set(CMAKE_ENABLE_WERROR ON)
-
-# Set the default build type to release with debug info
-if (NOT CMAKE_BUILD_TYPE)
-  set(CMAKE_BUILD_TYPE RelWithDebInfo
-    CACHE STRING
-      "Choose the type of build, options are: None Debug Release RelWithDebInfo MinSizeRel."
-  )
-endif (NOT CMAKE_BUILD_TYPE)
+set(CMAKE_ENABLE_WERROR ON)
```

Comment:
Unrelated changes

Reply:
The fact that I changed this is a vestige of the earlier logic I used, so, yes, I could remove it.